docs/compendium/substitute.md

```diff
@@ -25,7 +36,90 @@ introduction...
 ### Error responses
 
 ```json
+401 Unauthorized
+
+{"error":"not authenticated or not allowed"}
+```
+
+```json
+404 Not Found
+
+{"error":"compendia ids not found"}
+```
+
+### Request
+
+`POST /api/v1/substitution`
+
+input of request-body for substitution <br/>
+( where `checked` will be a list of files choosen for substitution )
+```json
+{
+  "user": "user_id",
+  "ercs": [
+    {
+      "id": "base_erc_id",
+      "checked": [],
+      "user": "creator_id"
```

Comment:
Why is this ID used for? The server can easily retrieve this information from the database.